wpilibj/src/main/java/edu/wpi/first/wpilibj/drive/KilloughDrive.java

```diff
@@ -240,7 +245,7 @@ public void drivePolar(double magnitude, double angle, double zRotation) {
    *     positive.
    * @param gyroAngle The current angle reading from the gyro in degrees around the Z axis. Use this
    *     to implement field-oriented controls.
-   * @return Wheel speeds.
+   * @return Wheel speeds [-1.0..1.0].
    */
   @SuppressWarnings("ParameterName")
   public WheelSpeeds driveCartesianIK(
```

Comment:
This method is not static in Java, nor in C++.

Reply:
That's because it relies on member variables that are assigned in the constructor.

Reply:
Ah, got it. Thanks!